.scala-steward.conf

```diff
@@ -0,0 +1,103 @@
+# pullRequests.frequency allows to control how often or when Scala Steward
+# is allowed to create pull requests.
+#
+# Possible values:
+#   @asap
+#     PRs are created without delay.
+#
+#   <timespan>
+#     PRs are created only again after the given timespan since the last PR
+#     has passed. Example values are "36 hours", "1 day", or "14 days".
+
+#   <CRON expression>
+#     PRs are created roughly according to the given CRON expression.
+#
+#     CRON expressions consist of five fields:
+#     minutes, hour of day, day of month, month, and day of week.
+#
+#     See https://www.alonsodomin.me/cron4s/userguide/index.html#parsing for
+#     more information about the CRON expressions that are supported.
+#
+#     Note that the date parts of the CRON expression are matched exactly
+#     while the time parts are only used to abide to the frequency of
+#     the given expression.
+#
+# Default: @asap
```

Comment:
Hmmm it seems that this setting does **not** affect the runs of Scala Steward itself... only just how often the PRs are created. Creating PRs is just a minor part of the workload for Scala Steward for this repo... the majority is probably checking for dependency updates themselves.

I'd leave the default `@asap` here... I think non-`@asap` settings only make sense in repos where there would be a constant flood of new PRs otherwise. This is clearly **not** the case for this project: we haven't got a single update from Scala Steward in ~2 weeks now, since none of our dependencies has released a new version recently.